src/raftstore/store/metrics.rs

```diff
@@ -61,4 +61,11 @@ lazy_static! {
             "Total number of raftstore snapshot traffic.",
             &["type"]
         ).unwrap();
+
+    pub static ref PEER_RAFT_PROCESS_NANOS_COUNTER_VEC: CounterVec =
```

Comment:
Why not `HistogramVec`? We can set the unit to `ns`, and histogram has the sum too.